python_benchmarks/normalization.py

```diff
@@ -47,16 +47,17 @@ def norm_fwd_fusion(
         weight = fd.ops.cast(weight, dtype=DataType.Float)
         bias = fd.ops.cast(bias, dtype=DataType.Float)
 
-    var, mean = fd.ops.var_mean(input, axes=reduction_axes, correction=0, keepdim=False)
```

Comment:
Isn't this PR supposed to be just about the validation?

Reply:
This was an error in the fusion definition due to the order of operations which was causing the validation tests to fail for normalization benchmarks. Should we have this in a separate PR?